contrib/bind/template.py

```diff
@@ -0,0 +1,40 @@
+pkgname = "bind"
+pkgver = "9.18.21"
+pkgrel = 0
+build_style = "gnu_configure"
+make_cmd = "gmake"
+make_dir = "."
+hostmakedepends = [
+    "automake",
+    "doxygen",
+    "gmake",
+    "libtool",
+    "pkgconf",
+    "python-sphinx",
+    "xsltproc",
+]
+makedepends = [
+    "cmocka-devel",
+    "heimdal-devel",
+    "libcap-devel",
+    "libuv-devel",
+    "libxml2-devel",
+    "json-c-devel",
+    "nghttp2-devel",
+    "openssl-devel",
+]
+checkdepends = ["python-pytest", "python-dnspython"]
+pkgdesc = "ISC DNS server"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "MPL-2.0"
+url = "https://www.isc.org/bind"
+source = f"https://downloads.isc.org/isc/bind9/{pkgver}/bind-{pkgver}.tar.xz"
+sha256 = "a556be22505d9ea4f9c6717aee9c549739c68498aff3ca69035787ecc648fec5"
+
+
+def post_install(self):
```

Comment:
it would be nice to make a -progs that separates the debugging cli utilities (and depend on it in the parent)

see https://pkgs.alpinelinux.org/contents?file=&path=&name=bind-tools&branch=edge&arch=x86_64 for a list of these

(mostly for dig but also the others just because)

Reply:
and not the sbin ones from there

Reply:
that seems fine